model/config.go

```diff
@@ -2430,6 +2430,27 @@ func (o *Config) ToJson() string {
 	return string(b)
 }
 
+func ConfigToJsonWithoutEmptyFields(o *Config) string {
```

Comment:
I want to handle the case when `o == nil`, that's why this is not a method on `Config`. That happens when plugins don't define a `RequiredConfig` section

Reply:
It's actually safe to call `ConfigToJsonWithoutEmptyFields` on a `nil` `Config`, so long as you don't dereference `o` -- something you already check, of course.

That being said, I'm inclined to recommend we move this to the `plugin` package (see later comment re: the `utils` package method), and perhaps rename to something like `DescribeRequiredConfig`. That way, plugin authors could use this to describe their own errors.